devices/datek.js

```diff
@@ -151,4 +151,26 @@ module.exports = [
             exposes.enum('service_mode', ea.ALL, ['deactivated', 'random_pin_1x_use',
                 'random_pin_24_hours']).withDescription('Service Mode of the Lock')],
     },
+    {
+        zigbeeModel: ['Water Sensor'],
+        model: 'HSE2919E',
+        vendor: 'Eva',
```

Comment:
I suggest to just put Datatek here and change the description to `Eva water leak sensor`.

Reply:
Done :)